### ISSUE TITLE
Add more customizable components

### ISSUE THREAD

**Issue by riboher** [gpbl/react-day-picker#1173]
Hi there!

This is probably not a bug but just a consequence of working with the beta version, so sorry in advanced if this is going to be fixed in further releases.

I'm trying to customise the caption section so I can append my own next/prev icons and build a different layout (I need the arrows on each side and the current month in the middle).

I see there are several functions that would help me achieve that (`IconNext`, `IconPrev` or `Caption`), but these components are not being exported so they can be used in the component prop of `DayPicker` component (like `Head`, `Row`, etc)

So far I've done something like this, even though I think that `Head` is not what I need
```javascript
const CustomCaptionLayout = () => {
  const { month, setMonth } = useNavigation();
  return (
    <>
      <button onClick={() => setMonth(month)}>prev</button>
      <p>{month}</p>
      <button onClick={() => setMonth(month)}>next</button>
    </>
  );
};

<DayPicker
  components={{
    Head: CustomCaptionLayout,
  }}
/>
```

Is this something that will be added in future releases or is there an appropriate reason for these components to not be exported like the rest?

Any idea how this could be achieved right now?

Thank you!

**Comment by gpbl** [comment]
Hey thanks for trying the beta. How was the experience so far? Was it hard to get the code? I'm sorry I've been changing lot of stuff (it should be called alpha) :) 

Yeah those components cannot be changed yet, but they should be trivial to implement, let see if I can add that to the next release. Definitely is a use case we need to cover!

**Comment by gpbl** [comment]
@riboher for moving the buttons on the sides and putting the caption at its center, you can easily override the default CSS:

<img width="1138" alt="Screenshot 2021-02-25 at 13 53 16" src="https://user-images.githubusercontent.com/120693/109202497-0db66880-7771-11eb-93a6-ff910c9dd71c.png">

**Comment by riboher** [comment]
Hey @gpbl, thanks for the quick reply :D.

Regarding the ease of use, we're actually migrating from react-dates so everything's a bit messed up, but the API seems clear enough and I got it working pretty fast.

The most challenging part is being mapping all the classnames to the components I need to style on my own, since I cannot use the default styles for the project I'm working on.

I'll try your CSS solution for the time being. The things is I wouldn't want to rely on absolute positioning if I can provide my own layout (also, I need to provide my own icons so it's something I'll need to change either way via the components prop)

Anyway, it's being great so far and I'm looking forward to the definitive version :D 
Thanks again!

**Comment by gpbl** [comment]
@riboher  just published `v8.0.0-beta.13` - there are [new components](https://react-day-picker-next.netlify.app/api/interfaces/components) to customize the icons.

> The most challenging part is being mapping all the classnames to the components I need to style on my own, since I cannot use the default styles for the project I'm working on.

Thanks for the feedback – for which reasons is that part challenging? Are you using CSS modules? I'd be interested to know how to improve this part. (I haven't tested it carefully yet).

> The things is I wouldn't want to rely on absolute positioning if I can provide my own layout 

I'd be curious to see your solution :)

DayPicker also displays the caption centered when displaying multiple months, no idea how to make it without absolute positioning _and_ keeping the label centered:

<img width="659" alt="Screenshot 2021-02-26 at 17 59 01" src="https://user-images.githubusercontent.com/120693/109364235-5139d100-785c-11eb-8d1b-453c36ddc7a4.png">

If you are coming up with a good solution we could add a `captionLayout="buttons-separe"` as alternative layout!
